docker/py3-tools/pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "~3.10"
 cloudshare = "*"
-intezer-sdk = "*"
+intezer-sdk = "<=1.19.9"
```

Comment:
you will need to add to dependabot config to ignore this. please add link to the issue as a comment here

Reply:
we sjould also lock native image